homeassistant/components/tuya/__init__.py

```diff
@@ -243,3 +243,32 @@ def async_update_entry() -> None:
             self.hass.config_entries.async_update_entry(self.entry, data=data)
 
         self.hass.add_job(async_update_entry)
+
+
+# Workaround for https://github.com/home-assistant/core/issues/151239
```

Comment:
We want to avoid doing workarounds in Home Assistant.

Let's handle this upstream.


../Frenck